PPOCRLabel.py

```diff
@@ -3665,6 +3679,89 @@ def expandSelectedShape(self):
             self.updateBoxlist()
             self.setDirty()
 
+    def resortBoxposion(self):
+        # Sort from top to bottom, left to right
+        def sort_rectangles(rectangles, row_height_threshold=0.5):
```

Comment:
`sort_rectangles` is generic, consider moving it outside so the method isn’t 100+ lines long.